lib/App/perlimports/Document.pm

```diff
@@ -400,12 +401,40 @@ sub _build_possible_imports {
 
         next if !$isa_symbol && is_method_call($word);
 
-        # A hash key might, for example, be a variable.
+        # The => operator (sometimes pronounced "fat comma") is a synonym for
+        # the comma except that it causes a word on its left to be interpreted
+        # as a string if it begins with a letter or underscore and is composed
+        # only of letters, digits and underscores. This includes operands that
+        # might otherwise be interpreted as operators, constants, single number
+        # v-strings or function calls.
+        # https://perldoc.perl.org/perlop#Comma-Operator
+        my $word_is_string;
+        if (   $word->statement->isa('PPI::Statement::Expression')
+            && $word->isa('PPI::Token::Word')
+            && $word->content =~ /^[a-zA-Z_][a-zA-Z0-9_]*$/ ) {
+            my @children = $word->statement->children;
+            while ( my $current = shift @children ) {
+                last unless refaddr($current) == refaddr($word);
+            }
+            while ( my $current = shift @children ) {
+                next if $current->isa('PPI::Token::Whitespace');
+                $word_is_string = 1
+                    if $current->isa('PPI::Token::Operator')
+                    && $current->content eq '=>';
+                last;
```

Comment:
If we've removed the whitespace above, do we even need the `while` loop here anymore? That may also reduce the complexity score.